### ISSUE TITLE
Change to command logic for compatibility with LuckPerms

### ISSUE THREAD

**Issue by authvin** [PG85/OpenTerrainGenerator#438]
When running OTG on SpongeForge alongside LuckPerms, neither commands nor the O menu work as intended, because LuckPerms "disables" the OP system. The game sees everyone as OP, and LuckPerms instead makes everything rely on permissions. As OTG isn't designed to fit in with Sponge permissions, this fails, and the GUI opens for any player on the server.

Sponge handles permissions by taking the command name string in a canCommandSenderUseCommand(int permlevel, String commandName) and making it a permission. So, if we sent "openterraingenerator.openui" as a commandname in the code that opens the ui, then vanilla Forge still only looks at the permLevel int, but Sponge now checks to see the user has the permission given as CommandName. Right now the commandName string is just the name of the command (like "map" or "biomes"), whereas with this change it'd be "openterraingenerator.map" or something along those lines. Again, as far as I'm aware this should not change Forge behaviour at all, and should make it compatible with LuckPerms.

[More info here, from Sponge](https://docs.spongepowered.org/stable/en/plugin/permissions.html#for-forge-mods)

Normally I would change this myself and submit a pull request, but I'm without a PC for a few days so I'm submitting the idea as a ticket instead for now.

TL;DR: Change how the names of commands are passed to fit the Sponge standard

**Comment by SXRWahrheit** [comment]
This is unlikely to be implemented anywhere in the near future, because:

1) OTG does not officially support Sponge
2) This seems like a Luckperms problem

**Comment by authvin** [comment]
I know OTG doesn't officially support Sponge, but that doesn't mean we can't fix a simple compatibility error for a popular plugin. And I thought it was a LuckPerms problem too at first, but they follow Sponge standard just fine. If this is to be fixed, it has to be done from OTG's side. And like I said, I'm very much willing to implement this myself and submit a pull request. It's not a big change.

**Comment by SXRWahrheit** [comment]
> because LuckPerms "disables" the OP system

> I thought it was a LuckPerms problem

I mean it still sounds like a LuckPerms problem.

**Comment by authvin** [comment]
The incompatibility comes from OTG not following Sponge permission standards. Considering how easy it is to comply with that standard, I'd say this is an OTG problem, not a LuckPerms one.

**Comment by PurpleIsEverything** [comment]
This is an issue with OTG not following Forge permission standards not Sponge or LuckPerms. 

LuckPerms is simply forcing a low op permission level.

Checking for OP status doesn't necessarily mean that a user has permission as there is an OP permission level specified in the server.properties file hence the presence of permLevel parameter on canCommandSenderUseCommand

**Comment by authvin** [comment]
#441 should fix this
